drug-interaction/README.md

```diff
@@ -0,0 +1,41 @@
+### Drug-drug interaction
+
+
+## Problem
+In this example, we attempt to infer unknown DDIs from a network of
+multiple drug-based similarities and known interactions. 
+
+## Dataset
+Each dataset below  contain seven drug–drug similarities. Four of these
```

Comment:
I think this section needs a little restructuring.
I would move the different dataset discussion to the first part, and then after that take a small paragraph to talk about the similarities.